### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by ainfeld**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/analytics-events-update.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+**Description**: `<Reason for update>`
+
+### Updates needed
+`<Event name>`
+- Current: `<Description of how the event works currently>`
+- Requested change: `<Description of the update needed>`
+
+#### People to cc
+**PM**: `<tag PM for the tool/feature>`
+**DS**: @ainfeld
```
[the remark this answers is not in the document]
Comment (ainfeld) [reply .github/ISSUE_TEMPLATE/analytics-events-update.md:23]: Would that be confusing if it's not a request for "work" from this person? If that's not confusing to how the structure is now then yes, but I would want to keep a note in the template that the PM should be added as an assignee

[the remark this answers is not in the document]
Comment (maniarathi) [reply .github/ISSUE_TEMPLATE/analytics-events-update.md:23]: I don't think PM is the assignee.. an engineer eventually will have to complete the code changes but PM should be in the loop.

File: .github/ISSUE_TEMPLATE/---feature-request.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 name: "\U0001F4A1 Feature request"
 about: Suggest an idea for this project
 title: "[FEATURE REQUEST]"
-labels: user request
+labels: ''
```
Comment (maniarathi) [inline .github/ISSUE_TEMPLATE/---feature-request.md:5]: Why this change?

File: .github/ISSUE_TEMPLATE/analytics-events-update.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ainfeld) [reply .github/ISSUE_TEMPLATE/analytics-events-update.md:23]: then yeah I would keep it as "cc'ing" them rather than adding them as an assignee

File: .github/ISSUE_TEMPLATE/---feature-request.md (hunk 2 above)
Comment (ainfeld) [reply .github/ISSUE_TEMPLATE/---feature-request.md:5]: Oh that might have been a mistake. I didn't mean to touch any of the other templates

**Comment by maniarathi** [comment]
@ainfeld If you make the change to undo that label change in `.github/ISSUE_TEMPLATE/---feature-request.md`, I will approve and you can merge!

File: .github/ISSUE_TEMPLATE/---feature-request.md (hunk 2 above)
Comment (maniarathi) [inline .github/ISSUE_TEMPLATE/---feature-request.md:5]:
```suggestion
labels: 'user request'
```

File: .github/ISSUE_TEMPLATE/---feature-request.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 name: "\U0001F4A1 Feature request"
 about: Suggest an idea for this project
 title: "[FEATURE REQUEST]"
-labels: user request
+labels: 'user request'
```
Comment (maniarathi) [inline .github/ISSUE_TEMPLATE/---feature-request.md:5]:
```suggestion
labels: user request
```

**Review by maniarathi — APPROVED** [review]
